scripts/setup-k8s-master.sh.in

```diff
@@ -76,20 +65,51 @@ bootstrapTokens:
   usages:
   - signing
   - authentication
-controllerManagerExtraArgs:
-  cloud-provider: "aws"
-  allocate-node-cidrs: "false"
+nodeRegistration:
+  criSocket: /var/run/dockershim.sock
+  name: "${HOSTNAME}"
+  kubeletExtraArgs:
+    cloud-provider: "aws"
+  taints:
+  - effect: NoSchedule
+    key: node-role.kubernetes.io/master
```

Comment:
The taints specified here should be the default and not need to be specified explicitly